spaceflights-pandas/hooks/post_gen_project.py

```diff
@@ -27,5 +27,5 @@ def main(selected_tools):
 
     # Execute the script only if the tool is selected.
     # This ensures the script doesn't run with kedro new --starter but only with the tools flow option.
-    if selected_tools != "none":
+    if selected_tools != "['None']":
```

Comment:
why we need this change?

Reply:
As part of the changes in Kedro#3426 to store the selected tools as a list instead of a string, a selection of no tools now results in the string 'None' being stored. As a result, the condition by which the hook checks if any tools were selected (or rather, were not selected) had to be updated as well.

Reply:
Thank you, missed it! But looks like it's not always true. If you prompted tools or put them from CLI value will be different. I will take some time to check

Reply:
Looks like we will kill starter with that change

Reply:
Yes, looks like that solution will heal "kedro new --starter=spaceflights-pandas", but "kedro new --tools="none"" will continue to work wrong. We should unify that situation with tools= ['None'] or '[]' first of all.

Reply:
This is unified in the latest version of https://github.com/kedro-org/kedro/pull/3462 where a selection of none or no tools will always result in `[None]` now